Source/Modules/ModuleRender.cpp

```diff
@@ -192,29 +193,35 @@ update_status ModuleRender::Update()
 	opaqueGOToDraw.clear();
 	transparentGOToDraw.clear();
 
-	const Skybox* skybox = App->GetModule<ModuleScene>()->GetLoadedScene()->GetSkybox();
+	ModuleWindow* window = App->GetModule<ModuleWindow>();
+	ModuleCamera* camera = App->GetModule<ModuleCamera>();
+	ModuleDebugDraw* debug = App->GetModule<ModuleDebugDraw>();
+	ModuleScene* scene = App->GetModule<ModuleScene>();
+	ModulePlayer* player = App->GetModule<ModulePlayer>();
+
+	const Skybox* skybox =scene->GetLoadedScene()->GetSkybox();
 	if (skybox)
 	{
 		skybox->Draw();
 	}
 
-	if (App->GetModule<ModuleDebugDraw>()->IsShowingBoundingBoxes())
+	if (debug->IsShowingBoundingBoxes())
 	{
-		DrawQuadtree(App->GetModule<ModuleScene>()->GetLoadedScene()->GetRootQuadtree());
+		DrawQuadtree(scene->GetLoadedScene()->GetRootQuadtree());
 	}
 
 	int w, h;
-	SDL_GetWindowSize(App->GetModule<ModuleWindow>()->GetWindow(), &w, &h);
+	SDL_GetWindowSize(window->GetWindow(), &w, &h);
 
-	App->GetModule<ModuleDebugDraw>()->Draw(App->GetModule<ModuleCamera>()->GetCamera()->GetViewMatrix(),
-		App->GetModule<ModuleCamera>()->GetCamera()->GetProjectionMatrix(), w, h);
+	debug->Draw(camera->GetCamera()->GetViewMatrix(),
+		camera->GetCamera()->GetProjectionMatrix(), w, h);
 
-	GameObject* goSelected = App->GetModule<ModuleScene>()->GetSelectedGameObject();
+	GameObject* goSelected = scene->GetSelectedGameObject();
 
 	bool isRoot = goSelected->GetParent() == nullptr;
 
-	FillRenderList(App->GetModule<ModuleScene>()->GetLoadedScene()->GetRootQuadtree());
-	std::vector<GameObject*> nonStaticsGOs = App->GetModule<ModuleScene>()->GetLoadedScene()->GetNonStaticObjects();
+	FillRenderList(scene->GetLoadedScene()->GetRootQuadtree());
```

Comment:
Maybe we could take this chance to remove some of this other repetitive calls, not only to the modules. There are a few examples, I'll point out some of them.
```suggestion
    Scene* loadedScene = scene->GetLoadedScene()
	FillRenderList(loadedScene->GetRootQuadtree());
```